docs/data/date-pickers/date-picker/ViewsDatePicker.js

```diff
@@ -12,11 +12,13 @@ export default function ViewsDatePicker() {
         <NextDatePicker
           label="Year only"
           views={['year']}
+          openTo="year"
           defaultValue={dayjs('2022-04-07')}
         />
         <NextDatePicker
           label="Year and Month"
           views={['year', 'month']}
+          openTo="year"
```

Comment:
Should not be needed after #7097, but I can remove it everywhere once we have the fix

Reply:
Gonna be even better after that. 👌